eng/pipelines/templates/jobs/tools-cleanup.yml

```diff
@@ -0,0 +1,32 @@
+parameters:
+- name: Repos
+  type: object
+  default:
+    - azure-sdk-for-android
+    - azure-sdk-for-c
+    - azure-sdk-for-cpp
+    - azure-sdk-for-go
+    - azure-sdk-for-ios
+    - azure-sdk-for-java
+    - azure-sdk-for-js
+    - azure-sdk-for-net
+    - azure-sdk-for-python
+
+jobs:
+  - job: CleanUp
+    pool:
+      vmImage: windows-2019
+    steps:
+      - ${{ each repo in parameters.Repos }}:
+        - task: PowerShell@2
+          displayName: Clean Up Sync Common Branches
+          condition: succeeded()
+          inputs:
+            pwsh: true
+            workingDirectory: $(System.DefaultWorkingDirectory)
+            filePath: $(System.DefaultWorkingDirectory)/eng/common/scripts/Delete-RemoteBranches.ps1
+            arguments: >
+              -RepoName ${{ repo }}
```

Comment:
Looks like you are not passing RepoOwner. I assume you plan to pass azure-sdk as these are cleaning up branches in the forks. 

We should also consider doing a similar branch clean-up for the increment version branches.